OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -880,7 +885,8 @@ public void RemoveInfluence()
 
 		public bool HasInfluence()
 		{
-			return landingCells.Any() || self.World.Map.DistanceAboveTerrain(CenterPosition).Length < Info.MinAirborneAltitude;
+			var altitude = self.World.Map.DistanceAboveTerrain(CenterPosition).Length;
```

Comment:
why waste a variable? It should be just within the return

Reply:
Just to don't overflow the line. I think the compiler will optimize something like this.